src/Raven.Client/Documents/Operations/Replication/DetailedReplicationHubAccess.cs

```diff
@@ -19,6 +20,10 @@ internal static string[] Preferred(string[] a, string[] b)
                 return a;
             return b ?? Array.Empty<string>();
         }
+    }
 
+    public class ReplicationHubAccessResponse
```

Comment:
internal (right?) and change `Response => Result`

Reply:
1. changing to internal
2. `ReplicationHubAccessResult` is already used in  
https://github.com/ravendb/ravendb/blob/dd4d680c333b5a72be837f47c21e1f91b92729d8/src/Raven.Client/Documents/Operations/Replication/ReplicationHubAccessResult.cs#L3